core/capabilities/webapi/webapicap/gen.go

```diff
@@ -0,0 +1,5 @@
+package webapicap
```

Comment:
should this be the package for all the webapi capabilities, notably the trigger and the target?  We seem to have 3 different package names in the handler now.
webapicap
trigger
target
And also webapicapabilities in gateway.

Reply:
I don't know who uses webapicapabilities, but if it's only for these capabilities, it should have the duplicated types removed in favour of these ones.

webapicap should hold the wasm builders for both the trigger and the target IMO.

The trigger and target can either be in the webapi folder, or subfolders called trigger and target.  I'm less opinonated about that one.  @DeividasK has probably put more thought into it.